demos/kqueen_azure-demo.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
+
+################################################
+# include the demo-magic script
+################################################
+# demo-magic.sh is a handy shell script that enables you to script repeatable demos in a bash environment so you don't have to type as you present
+# Rather than trying to type commands when presenting you simply script them and let demo-magic.sh run them for you
+test -f ./demo-magic.sh || curl --silent https://raw.githubusercontent.com/paxtonhare/demo-magic/master/demo-magic.sh > demo-magic.sh
+. ./demo-magic.sh -n
+
+################################################
+# Configure the options
+################################################
+
+#
+# speed at which to simulate typing. bigger num = faster
```

Comment:
Do you have a more specific description? E.g. it's symbols per minute or smth like that